lib/rules/quotes.js

```diff
@@ -28,6 +40,7 @@ export default class Quotes extends Rule {
       [
         '  * "double" - requires the use of double quotes wherever possible',
         '  * "single" - requires the use of single quotes wherever possible',
+        '  * { hbs: `single`|`double`, html: `single`|`double` } - requires different quotes for Handlebars and HTML syntax',
```

Comment:
Regarding the names `hbs` and `html`, can you explain a bit more how you came to those names?

Reply:
It's just how I see the difference, I didn't really think much about it as I was in a rush and pretty tired. I don't know what they should be called.

Reply:
The hbs file extension doesn't exist in strict mode, so i wonder if this should be 'expressions'?
- expressions
- template expressions
- curlies
- within curlies
- within curly braces
- within expressions 

Ond then:
- within html
- outside expressions
- outside curly braces

?

Idk

Reply:
What about `html` and `mustache`? Does that accurately describe the difference?

Reply:
I think `html` and `curlies` would be good. Mainly because `curlies` is a term used a fair bit with ember.

Been pretty busy so haven't gotten back to this but should be good over the weekend or early next week.

Reply:
That's fair, there are two ember-template-lint rules with `curlies` in the name, for prior art. Let's proceed with that then and get this merged in the coming days!